getting_started/editor/unity_to_godot.rst

```diff
@@ -259,7 +259,8 @@ Creating a script that inherits :ref:`class_Resource` will allow you to create c
 Using Godot in C++
 ------------------
 
-For your information, Godot also allows you to develop your project directly in C++ by using its API, which is not possible with Unity at the moment. As an example, you can consider Godot Engine's editor as a "game" written in C++ using Godot API.
+Godot allows you to develop your project directly in C++ by using its API, which is not possible with Unity at the moment. 
+As an example, you can consider Godot Engine's editor as a "game" written in C++ using the Godot API.
```

Comment:
Maybe re-word to something like "As an example, the Godot editor is essentially a game written in C++ using the Godot API".

Reply:
Hmmm, I didn't want to use "essentially" or "basically" but I see what you're getting at! I was considering the same idea.

How about "In fact, Godot Engine's editor is practically a "game" written in C++ using the Godot API."?

As in, "in practice" it is a game.